pulser/tests/test_simulation.py

```diff
@@ -326,6 +326,9 @@ def test_config():
     sim = Simulation(seq, sampling_rate=0.01)
     sim.reset_config()
     sim.show_config()
+    with pytest.raises(ValueError, match="not a valid"):
+        sim.set_config('bad_config')
+    sim.update_config(SimConfig(noise=('SPAM', 'doppler', 'amplitude')))
```

Comment:
Instead of just making the lines run, you should check that it's doing what it's supposed to by accessing the values in `SimConfig`.